zmsstatistic/src/Zmsstatistic/Download/RequestReport.php

```
@@ -95,6 +95,13 @@ public function writeReportData(ReportEntity $report, $sheet)
         foreach ($report->data as $name => $entry) {
             if ($name !== 'sum' && $name !== 'average_processingtime') {
                 $rowData = [];
+                if ($name === 'uncategorized') {
+                    $rowData[] = 'Dienstleistung wurde nicht erfasst';
+                } elseif ($name === 'nonexistent') {
+                    $rowData[] = 'Dienstleistung konnte nicht erbracht werden'
+                } else {
```

PHPStan (2.2.1)

[error] Syntax error, unexpected '}' on line 102


(phpstan.parse)